README.md

```diff
@@ -71,7 +71,7 @@ Some configuration parameters can be changed with environment variables:
    - `SD_TEMPLATE_DIR`: when using SD_CONFIG_BACKEND, set the path where the check configuration templates are located in the key-value store (default is `datadog/check_configs`)
    - `SD_CONSUL_TOKEN`: when using Consul as a template source and the Consul cluster requires authentication, set a token so the Datadog Agent can connect.
    - `SD_BACKEND_USER` and `SD_BACKEND_PASSWORD`: when using etcd as a template source and it requires authentication, set a user and password so the Datadog Agent can connect.
-   - `SD_JMX_ENABLE`: set to `yes` to enable Autodiscovery for JMX-based checks. **Use with `docker-dd-agent:latest-jmx`; this image includes a JVM, which is needed to run jmxfetch.**
+   - `SD_JMX_ENABLE`: set to `yes` to enable Autodiscovery for JMX-based checks. JMX images (e.g. `latest-jmx`) automatically set this to `yes` in datadog.conf, so you don't need to pass it in when running those images.
```

Comment:
Actually, since SD_JMX_ENABLE would only ever be useful with JMX images, it is even useful to document this option? I suppose it's useful for those who want to disable it, i.e. SD_JMX_ENABLE=no, though will many users want to do that?